tutorials/plot_sensors_time_frequency.py

```diff
@@ -70,12 +72,103 @@
 
 ax.plot(freqs, psds_mean, color='k')
 ax.fill_between(freqs, psds_mean - psds_std, psds_mean + psds_std,
-                color='k', alpha=.5)
+                color='k', alpha=.15, lw=0)
 ax.set(title='Multitaper PSD (gradiometers)', xlabel='Frequency',
        ylabel='Power Spectral Density (dB)')
 plt.show()
 
 ###############################################################################
+# To see the difference between :func:`psd_multitaper` and :func:`psd_welch`
+# - we are going to compare PSDs obtained with each function.
+ch_index = epochs.ch_names.index('MEG 2333')
```

Comment:
That's a tutorial actually - I thought it should be a little longer. The psd was applied to epochs in this tutorial before - so I just added combine to what already was present. Performing psd on raw will take longer but the effects could be more visible - should I change it?
I'll add `combine` to plot_psd.